CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 # Braintree Android SDK Release Notes
 
+## unreleased
+
+* Add `BraintreeNonce` result type for `GetPaymentMethodNoncesCallback`
```

Comment:
Nitpick / take it or leave it

```suggestion
* Replace `PaymentMethodNonce` with `BraintreeNonce` result type for `GetPaymentMethodNoncesCallback`
```

Reply:
Removed this line since we removed `BraintreeNonce`